engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -442,14 +406,14 @@ public void renderWorld(Camera camera) {
         DefaultRenderingProcess.getInstance().beginRenderLightGeometryStencilPass();
         Material program = Assets.getMaterial("engine:prog.simple");
```

Comment:
There is actually no "engine:simple" material asset. There is an "engine:simple" shader. The lwjgl shader manager generates materials for a number of shaders used as part of the rendering, and to ensure these materials don't conflict with any in the assets folders their names are prefixed with "prog."